build/gulp/tasks/test-projects.tsx

```diff
@@ -120,6 +125,54 @@ export default App;
 
     log('Test project is built successfully!')
   } finally {
-    fs.unlinkSync(stardustPackageFilename)
+    fs.unlinkSync(packageFilename)
   }
 })
+
+task('test:projects:rollup', async () => {
+  const logger = debug('bundle:rollup')
+  logger.enabled = true
+
+  const packageFilename = createPackageFilename()
+
+  await buildAndPackStardust(packageFilename)
+  logger(`✔️Stardust UI package was prepared: ${packageFilename}`)
+
+  const tmpDirectory = tmp.dirSync({ prefix: 'stardust-' }).name
+  logger(`✔️Temporary directory was created: ${tmpDirectory}`)
+
+  const dependencies = [
+    'rollup',
+    'rollup-plugin-replace',
+    'rollup-plugin-commonjs',
+    'rollup-plugin-node-resolve',
+    'react',
+    'react-dom',
+  ].join(' ')
+  await runIn(tmpDirectory)(`yarn add ${dependencies}`)
+  logger(`✔️Dependencies were installed`)
+
+  await runIn(tmpDirectory)(`yarn add ${packageFilename}`)
+  logger(`✔️Stardust UI was added to dependencies`)
+
+  fs.copyFileSync(
+    path.resolve(__dirname, '..', 'scaffold', 'app.js'),
+    path.resolve(tmpDirectory, 'app.js'),
+  )
+  fs.copyFileSync(
+    path.resolve(__dirname, '..', 'scaffold', 'rollup.config.js'),
+    path.resolve(tmpDirectory, 'rollup.config.js'),
+  )
+  logger(`✔️Source and bundler's config were created`)
+
+  await runIn(tmpDirectory)(`yarn rollup -c`)
+  logger(`✔️Example project was successfully built: ${tmpDirectory}`)
+})
+
+task(
+  'test:projects',
+  series(
```

Comment:
actually, could we run them in parallel? It seems that there is no shared state between these tasks

Reply:
I will left it as is for now. When this task will be enable, we will able to test that it works in parallel. However, I almost sure that it will work.